build_tools/travis/install.sh

```diff
@@ -37,15 +37,20 @@ if [[ "$DISTRIB" == "conda" ]]; then
     echo "Cached in $HOME/download :"
     ls -l
     echo
-    if [[ ! -f miniconda.sh ]]
-        then
-        wget http://repo.continuum.io/miniconda/Miniconda-3.6.0-Linux-x86_64.sh \
-            -O miniconda.sh
+    if [[ ! -f miniconda.sh ]]; then
+        if [[ "$PYTHON_VERSION" == "2.7" ]]; then
+            wget http://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh \
+                -O miniconda.sh
+            export PATH=/home/travis/miniconda2/bin:$PATH
+        else
+            wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+                -O miniconda.sh
+            export PATH=/home/travis/miniconda3/bin:$PATH
         fi
+    fi
     chmod +x miniconda.sh && ./miniconda.sh -b
     cd ..
-    export PATH=/home/travis/miniconda/bin:$PATH
```

Comment:
Do we still want `conda update -y conda`, instead of pinning to the version corresponding to the latest "official" release from Continuum?